### PR TITLE
Signalment final

### PR DESCRIPTION
**Pull request by Aditya-Jugran**

### CODE REVIEW

File: src/component/Signalment.js [hunk 1]
```diff
@@ -1,59 +1,78 @@
...
+class SignalmentForm extends React.Component {
 
-class SingnalmentForm extends React.Component {
-
-  onChange = (checked)=> {
-    console.log(`switch to ${checked}`);
+  constructor(props) {
+    super(props);
+    this.state = {
+      Distance:props.signalmentDetails[props.signalmentDetails.length-1] && props.signalmentDetails[props.signalmentDetails.length-1].Distance,
```
Comment (mageexxcel) [inline src/component/Signalment.js:14]: what is this ???? 
why array ?

File: src/component/Signalment.js [hunk 2]
```diff
@@ -1,59 +1,78 @@
...
+    this.state.own_car =  check;
   }
 
   handleSubmit = (e) => {
     e.preventDefault();
     this.props.form.validateFields((err, values) => {
-
       if (!err) {
-        console.log('Received values of form: ', values);
+         store.dispatch(usersignalmentDetails(values,this.state.driving_license,this.state.own_car));
```
Comment (mageexxcel) [inline src/component/Signalment.js:35]: change this is using  disptachtoprops which i told u

File: src/component/Signalment.js (hunk 1 above)
Comment (Aditya-Jugran) [reply src/component/Signalment.js:14]: signalmentDetails is an array so everytime when data is saved we need to access the last feeded data

File: src/component/Signalment.js (hunk 2 above)
Comment (Aditya-Jugran) [reply src/component/Signalment.js:35]: ok sir

**Pull request merged**